File: src/main/java/com/appconfig/dto/CheckLoginModel.java
package com.appconfig.dto;

public class CheckLoginModel {

    private boolean authenticated;
    private String user;

    public CheckLoginModel(boolean authenticated, String user) {
        this.authenticated = authenticated;
        this.user = user;
    }

    public boolean isAuthenticated() {
        return authenticated;
    }

    public void setAuthenticated(boolean authenticated) {
        this.authenticated = authenticated;
    }

    public String getUser() {
        return user;
    }

    public void setUser(String user) {
        this.user = user;
    }
}

File: src/main/frontend/src/app/login/checkLogin.model.ts
export class CheckLoginModel {

  private _authenticated: boolean;
  private _username: string;


  get authenticated(): boolean {
    return this._authenticated;
  }

  set authenticated(value: boolean) {
    this._authenticated = value;
  }

  get username(): string {
    return this._username;
  }

  set username(value: string) {
    this._username = value;
  }
}
